Samples/TardisBank/Solutions/Suteki.TardisBank.Tests/ServiceLocatorInitializer.cs

```diff
@@ -23,12 +19,6 @@ public static void Init()
                         .ImplementedBy(typeof(EntityDuplicateChecker))
                         .Named("entityDuplicateChecker"));
 
-            container.Register(
-                    Component
-                        .For(typeof(ISessionFactoryKeyProvider))
```

Comment:
SessionKey provider is not required now

Regards, Vitaliy
On Nov 25, 2015 7:11 PM, "Seif Attar" <notifications@github.com> wrote:

> In
> Samples/TardisBank/Solutions/Suteki.TardisBank.Tests/ServiceLocatorInitializer.cs
> <https://github.com/sharparchitecture/Sharp-Architecture/pull/125#discussion_r45933350>
> :
>
> > @@ -23,12 +19,6 @@ public static void Init()
> >                          .ImplementedBy(typeof(EntityDuplicateChecker))
> >                          .Named("entityDuplicateChecker"));
> >
> > -            container.Register(
> > -                    Component
> > -                        .For(typeof(ISessionFactoryKeyProvider))
>
> @shatl <https://github.com/shatl> what are those changes for nhibernate?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sharparchitecture/Sharp-Architecture/pull/125/files#r45933350>
> .
>